src/pages/Sources.js

```diff
@@ -139,6 +139,7 @@ const SourcesPage = () => {
   const mainContent = () => (
     <React.Fragment>
       <PrimaryToolbar
+        className="sources-toolbar"
```

Comment:
Can you please follow the naming style we are using? 

i.e. `ins-c-source__sources-toolbar`